### ISSUE TITLE
Add motion-safe animations to entering components

### ISSUE THREAD

**Issue by DerTimonius** [shadcn-ui/ui#1029]
First of all, great component library!

I think it's great that the components have been built with a11y in mind. The only I'm missing is the option for **reduced animations** since some users might prefer this setting.

A possible idea to achieve this would be adding the `motion-safe` class to certain components like `Dialog` or `Toast`.

**Comment by shadcn** [comment]
This issue has been automatically closed because it received no activity for a while. If you think it was closed by accident, please leave a comment. Thank you.
